cookbook/admin.py

```diff
@@ -163,10 +168,18 @@ def delete_unattached_steps(modeladmin, request, queryset):
 
 
 class StepAdmin(admin.ModelAdmin):
-    list_display = ('name', 'order',)
-    search_fields = ('name',)
+    list_display = ('recipe_and_name', 'order', 'space')
+    ordering = ('recipe__name', 'name', 'space', )
+    search_fields = ('name', 'recipe__name')
     actions = [delete_unattached_steps]
```

Comment:
usually admin actions affect only selected entries, with this we will delete everything from every space, do you think that is sufficently clear to the user?

I am leaning towards yes as playing in admin is always only for advanced users but wanted to check